### ISSUE TITLE
Voice is not playing over bluetooth

### ISSUE THREAD

**Issue by shyam3089** [MycroftAI/mimic#212]
**Describe the bug**

**To Reproduce**
4. See error '....'

**Expected behavior**

**Log files**
If possible, add log files from `/var/log/mycroft/` to help explain your problem.

You may also include screenshots, however screenshots of log files are often difficult to read and parse. 

If you are running Mycroft, the [Support Skill](https://github.com/MycroftAI/skill-support) helps to automate gathering this information. Simply say "Create a support ticket" and the Skill will put together a support package and email it to you.

**Environment (please complete the following information):**
 - Device type: [e.g. Raspberry Pi, Mark 1, desktop]
 - OS: [e.g. Ubuntu, Picroft]
 - Mycroft-core version: [e.g. 20.08]
 - Other versions: [e.g. Adapt v0.3.7]

**Additional context**

Please think carefully about whether you have modified anything in Mycroft's code or configuration files.  If so, can you reproduce this on a clean installation of Mycroft? Many "bugs" turn out to be non-standard configuration errors.

**Comment by shyam3089** [comment]
I have found this weird issue, I'm trying to create a voice assistant in raspberry pi and when I connect to a bluetooth speaker only the text to speech voice is not playing. I tested the connection by playing an mp3 file and it works fine(using pygame).

Mimic voice is playing If i switch audio output to analog (audio jack). 

This command works fine with analog output but I don't hear any sound with a bluetooth connection.

`/home/pi/mimic1/mimic --setf duration_stretch=0.75 --setf 'int_f0_target_stddev=20' --setf 'f0_shift=1' -voice /home/pi/mimic1/voices/cmu_us_rms.flitevox --setf 'int_f0_target_mean=120' -t 'hello, how are you'`

**Comment by forslund** [comment]
Hi @shyam3089 have you built mimic from source? and if so did you configure any specific audiobackend? It defaults to alsa, I wonder if it would be worth trying compiling with pulseaudio as audio backend?

**Comment by shyam3089** [comment]
Yeah I built it from source. I will try that. I had issues related to alsa. May be I messed up something.
